cupy/core/core.pyx

```diff
@@ -1508,6 +1508,9 @@ cdef class ndarray:
     def __str__(self):
         return str(self.get())
 
+    def __format__(self, format_spec):
+        return self.get().__format__(format_spec)
```

Comment:
For consistency around the lines, the builtin `format` looks better here, too.
```suggestion
        return format(self.get(), format_spec)
```

Do you have other pros/cons in mind?  e.g. performance

Reply:
I think that since this implies device synchronization, there is not much to care about. Thanks!